src/Stats/redux/reducer.jsx

```diff
@@ -0,0 +1,55 @@
+import {} from 'react-navigation-stack';
+
+import moment from 'moment';
+
+const initialState = {
+  transactions: [],
+  newTransType: 'Expense',
+  newTransDate: moment().format('DD/MM/YYYY'),
+  newTransAmount: null,
+  newTransInsertionSuccess: false,
+};
+
+export const actionType = {
+  ADD_TRANS: 'ADD_TRANS',
```

Comment:
action should be defined in the action creators

Reply:
fixed